app/models/mixins/supports_feature_mixin.rb

```diff
@@ -181,29 +65,19 @@ module SupportsFeatureMixin
     private_class_method :define_supports_feature_methods
   end
 
-  class UnknownFeatureError < StandardError; end
-
-  def self.guard_queryable_feature(feature)
-    unless QUERYABLE_FEATURES.key?(feature.to_sym)
-      raise UnknownFeatureError, "Feature ':#{feature}' is unknown to SupportsFeatureMixin."
-    end
-  end
-
   def self.reason_or_default(reason)
     reason.present? ? reason : _("Feature not available/supported")
```

Comment:
Here you might have to add the `_()` around reason so it's translating at access time.

Reply:
@chessbyte Do you still need to do this or is it unnecessary?

Reply:
the `reason` comes from `unsupported_reason_add` calls, which are already translated.